### PR TITLE
[PWA] Add images to event teams tab

### PR DESCRIPTION
**Pull request by tervay**
~Performance is pretty bad. Need to figure out improvements~

<details>
<summary>Screenshots</summary>

image

</details>

### CODE REVIEW

**Comment by fangeugene** [comment]
Can we do this with tanstack query on client side only

**Comment by tervay** [comment]
Performance issues are scrolling related, not network related

**Comment by tervay** [comment]
Performance issues resolved by loading the small image (`{}s.jpg`) rather than the full size image

File: pwa/app/routes/event.$eventKey.tsx [hunk 1]
```diff
@@ -416,3 +435,33 @@ function TeamsTab({
...
+
+    if (media.type === 'instagram-image') {
+      return `https://www.thebluealliance.com/instagram_oembed/${media.foreign_key}`;
+    }
+
+    return media.direct_url;
+  }
+
+  return maybePreferred === undefined
+    ? imageUrl(pics[0])
```
Comment (fangeugene) [inline pwa/app/routes/event.$eventKey.tsx:465]: I don't see a lot of value doing pics[0] here. I think if there aren't any preferred then we just return undefined.

File: pwa/app/routes/event.$eventKey.tsx [hunk 2]
```diff
@@ -375,11 +385,20 @@ function TeamsTab({
...
-                alt=""
-                className="size-20"
-              />
+              {(() => {
+                const maybeImage = getTeamRobotPic(
+                  media.filter((m) => m.team_keys.includes(t.key)),
+                );
+
+                return maybeImage === undefined ? null : (
+                  <img
```
Comment (fangeugene) [inline pwa/app/routes/event.$eventKey.tsx:394]: On prod this is a carousel. Do we want to replicate that?

Comment (tervay) [reply pwa/app/routes/event.$eventKey.tsx:394]: IMO we don't need to. I don't think it adds a ton of value. We could always add the carousel to the team-event modal.

**Pull request merged**